bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+```
+BIP: ?
+Title: Updated BIP Process
+Author: Murch <murch@murch.one>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-Updated-BIP-Process
+Status: Draft
+Type: Process
+Created: 2024-05-13
+License: BSD-2-Clause
+Requires: 123
+Replaces: 2
```

Comment:
While that’s the intended outcome, I think it would be more appropriate to set BIP 2 to superseded with the same pull request that sets this BIP to Active.